catalog/USGS/USGS_NLCD_RELEASES_2021_REL_TCC_v2021-4.jsonnet

```diff
@@ -0,0 +1,387 @@
+  ],
+  keywords: [
+    'forest',
+    'gtac',
+    'landsat_derived',
+    'redcastle_resources',
+    'sentinel2_derived',
+    'time_series',
+    'usda',
+    'usfs',
+    'usgs',
+  ],
+  providers: [
+    ee.producer_provider('USDA Forest Service (USFS) Geospatial Technology and Applications Center (GTAC)', 
+    'https://www.fs.usda.gov/about-agency/gtac'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent(-135.286387, 20.38379, -56.446306, 52.459364,
+                    '2008-06-01T00:00:00Z', '2021-09-30T00:00:00Z'),
+  summaries: {
+    'gee:schema': [
+      {
+        name: 'study_area',
+        description: |||
+          TCC currently covers CONUS, Southeastern Alaska, Puerto Rico-US
+          Virgin Islands and Hawaii. This version contains all study areas.
+          Possible values: 'CONUS, SEAK, PRUSVI, HI'
+        |||,
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'version',
+        description: |||
+          This is the fourth version of the TCC product released in the MRLC consortium 
+          that is part of the National Land Cover Database (NLCD)'
+        |||,
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'startYear',
+        description: |||
+          'Start year of the product'
+        |||,
+        type: ee_const.var_type.int,
+      },
+      {
+        name: 'endYear',
+        description: |||
+          'End year of the product'
+        |||,
+        type: ee_const.var_type.int,
+      },
+      {
+        name: 'year',
+        description: |||
+          'Year of the product'
+        |||,
+        type: ee_const.var_type.int,
+      },
+    ],
+    gsd: [30],
+    'eo:bands': [
+      {
+        name: 'Science_Percent_Tree_Canopy_Cover',
+        description: |||
+          The raw direct model outputs. Each pixel has a mean predicted tree canopy 
+          cover value for each year.
+        |||,
+        'gee:units': units.percent,
+      },
+      {
+        name: 'Science_Percent_Tree_Canopy_Cover_Standard_Error',
+        description: |||
+          The standard deviation of the predicted values from all regression trees we refer 
+          to as standard error. Each pixel has a standard error for each year.
+        |||,
+        'gee:units': units.percent,
+      },
+      {
+        name: 'NLCD_Percent_Tree_Canopy_Cover',
+        description: |||
+          To produce NLCD tree canopy cover, a post-processing workflow is applied to the direct 
+          model output that identifies and sets non-treed pixel values to zero percent 
+          tree canopy cover. NLCD tree canopy cover data are fully masked in 2008, 2009 and 2010. 
+        |||,
+        'gee:units': units.percent,
+      },
+      {
+        name: 'data_mask',
+        description: |||
+          Three values representing areas of no data, mapped tree canopy cover, and non-processing area.
```

Comment:
could you say more what "non-processing area" means?

Reply:
Sure thing, I'm adding in some additional info